development/scripts/fix-spec.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+import yaml
+
+SPEC_PATH = '/client/api/openapi.yaml'
+
+# Load the spec file
+with open(SPEC_PATH, 'r') as file:
+    data = yaml.load(file, Loader=yaml.CLoader)
+
+# Traverse schemas
+if 'components' in data and 'schemas' in data['components']:
+    for name, schema in data['components']['schemas'].items():
+        # Handle failover_strategy
+        if 'failover_strategy' in schema.get('properties', {}):
+            prop = schema['properties']['failover_strategy']
+            if 'default' in prop and prop['default'] is None:
+                print(f"Removing 'default: null' in {name}.failover_strategy")
+                del prop['default']
```

Comment:
For reference:

```yaml
        failover_strategy:
          type: object
          properties:
            value:
              type: string
              enum:
              - ''
              - active-active
              - active-passive
            label:
              type: string
              enum:
              - (unspecified)
              - Active/Active
              - Active/Passive
          default: null
```

Looks like the default should probably be the empty string, not null.

Reply:
Per slack it's the latter case that's an issue. Probably both are wrong though.

Reply:
Possibly relevant: https://github.com/OpenAPITools/openapi-generator/issues/16775

Reply:
Not clear yet where the `null` is coming from here as both the model and the serializer define `blank=True` and `allow_blank=True` but not `null=True` or `allow_null=True` here.

Reply:
So this is an issue of the specification, right?
Should I create an issue on nautobot's project?

Reply:
Created an issue :)
https://github.com/nautobot/nautobot/issues/6157